tests/integration/plots/test_plots.py

```diff
@@ -222,18 +212,27 @@ def test_repo_with_plots(
     assert html_result["linear.json"]["data"]["values"] == _update_datapoints(
         linear_v1,
         {
-            "rev": "workspace",
+            REVISION_FIELD: "workspace",
+            VERSION_FIELD: {
+                "revision": "workspace",
+                "filename": "linear.json",
+                "field": "y",
+            },
```

Comment:
Since the test is for VSCode integration, it probably makes more sense to test against the json result, right?

```python
json_result["linear.json"][0]["datapoints"]["workspace"] == _update_datapoints(
        linear_v1,
        {
            VERSION_FIELD: {
                "revision": "workspace",
                "filename": "linear.json",
                "field": "y",
            },
        },
)
```

Reply:
To be sure we need to check both, added.